frame/clearing-house/src/tests/mod.rs

```diff
@@ -136,7 +136,7 @@ fn get_position(account_id: &AccountId, market_id: &MarketId) -> Option<Position
 }
 
 fn get_outstanding_gains(account_id: AccountId, market_id: &MarketId) -> Balance {
```

Comment:
should this function be change to `get_outstanding_profits` to match the change from `outstanding_gains` to `outstanding_profits`?

Reply:
Yeah that makes sense. I forgot to go through the test files to change the naming there too